clients/java/src/main/java/marquez/client/MarquezClient.java

```diff
@@ -87,6 +88,24 @@ public MarquezClient(final URL baseUrl, @Nullable final String apiKey) {
     this.http = http;
   }
 
+  public List<RawLineageEvent> listEvents() {
```

Comment:
As discussed [here](https://github.com/MarquezProject/marquez/pull/2070#discussion_r950534505), we'll want to define this method as:

```
MarquezClient.listLineageEvents()
```

Reply:
Done.